step-web/src/main/webapp/js/backbone/views/view_main_search.js

```diff
@@ -235,20 +238,31 @@ var MainSearchView = Backbone.View.extend({
         }).on("select2-removed", function () {
             //re-evaluate master version
             self._reEvaluateMasterVersion();
+
+            // submit search automatically if we have more than just resources in the search
+            var searchData = self.masterSearch.select2("data");
+
+            if (!_.isEmpty(searchData)) {
```

Comment:
Added this to search when items are removed, as long as we have something other than versions. This prevents a loop if you delete Gen1, leaving only ESV. This search sets Gen1 as the passage. Round and round we go